platform/centec-arm64/docker-saiserver-centec/port_config.ini

```diff
@@ -0,0 +1,53 @@
+# name          lanes          alias             index     speed    fec
```

Comment:
do we really need this? i thought the saiserver will use the port_config.ini in the device template.

Reply:
I'll reference to PR #8625, reads config from hwsku folder.